mii/batching/ragged_batching.py

```diff
@@ -349,20 +347,45 @@ def _queue_flush_request(self, uid: int) -> None:
             ))
 
     def reset_request_status(self):
+        ## Get the last request that consumes KV cache
+        last_r = None
         for r in self.buffer:
             if r.seq_length > 0:
-                self._queue_flush_request(r.uid)
+                last_r = r
+        assert last_r is not None
```

Comment:
Can we add an error message here to indicate something has gone wrong?

Reply:
Thank you for the comment! I added a message